daisy/target_pool.go

```diff
@@ -0,0 +1,83 @@
+package daisy
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"regexp"
+
+	daisyCompute "github.com/GoogleCloudPlatform/compute-image-tools/daisy/compute"
+	"google.golang.org/api/compute/v1"
+	"google.golang.org/api/googleapi"
+)
+
+var (
+	targetPoolURLRegex = regexp.MustCompile(fmt.Sprintf(`^(projects/(?P<project>%[1]s)/)?regions/(?P<zone>%[2]s)/TargetPools/(?P<targetPool>%[2]s)$`, projectRgxStr, rfc1035))
```

Comment:
Isn't it `/targetPools/`? (lowercase)

Reply:
I thought the URL is case insensitive, but to make sure change to make it consistent with cloud document.